src/App/Tracker/Model/IssueModel.php

```diff
@@ -201,6 +204,7 @@ public function getItem($identifier)
 				->from($this->db->quoteName('#__issues_tests'))
 				->where($this->db->quoteName('item_id') . ' = ' . (int) $item->id)
 				->where($this->db->quoteName('result') . ' = 1')
+				->where($this->db->quoteName('sha') . ' = ' . $this->db->quote($lastCommit->sha))
```

Comment:
Looks like this send a error:

```
8 Whoops\Exception\ErrorException
Trying to get property of non-object
```

Example: https://trackertest-elkuku.rhcloud.com/tracker/jtests/90